web/book/src/reference/syntax/pipes.md

```diff
@@ -15,19 +15,20 @@ filter department == "Product"
 select {first_name, last_name}
 ```
 
-In the place of a line-break, it's also possible to use the `|` character to
+In the place of a line break, it's also possible to use the `|` character to
 pipe results, such that this is equivalent:
 
 ```prql
 from employees | filter department == "Product" | select {first_name, last_name}
 ```
 
-A line-break doesn't create a pipeline in a couple of cases:
+A line break doesn't create a pipeline in a few cases:
 
 - within a tuple
 - within an array
 - when the following line is a new statement, which starts with a keyword of
-  `func`, `let` or `from`.
+  `func`, `let` or `from`
```

Comment:
Is the reference to `func` still correct here or is this outdated since the introduction of `let`?

IIRC, a function definition including the optional `func` would need to be written as

```elm
let my_func = func x -> ...
```

So a line would never start with `func` anymore.

Reply:
I was actually just about to remove this! I was going to completely remove `func`...